java/java.source.base/src/org/netbeans/api/java/source/TreeMaker.java

```diff
@@ -1224,6 +1224,10 @@ public Tree BindingPattern(CharSequence name,
                                Tree type) {
         return delegate.BindingPattern(name, type);
     }
+    
+    public Tree BindingPattern(VariableTree vt) {
```

Comment:
It would be good to have `@since` flag here, and the module's specification version updated, + ideally apichanges.xml updated.
Also, possibly, deprecate the old API method above?

Reply:
Addressed code review comments.